tfhe/src/integer/server_key/radix_parallel/ilog2.rs

```diff
@@ -267,6 +397,33 @@ impl ServerKey {
         self.unchecked_leading_ones(ct)
     }
 
+    /// Returns the base 2 logarithm of the number, rounded down.
+    pub fn smart_ilog2<T>(&self, ct: &mut T) -> RadixCiphertext
```

Comment:
if we can redirect to a docstring with the proper example, even if it's not the smart one, same for other APIs

For the reminder of what smart/default ops are I think we will need to do a pass on the documentation to have it centralized and just refer to it whenever